docs/source/contributing.rst

```diff
@@ -227,40 +227,40 @@ with:
 
    make
 
-Any misspelled words will be reported.  You can add words to be
+Any misspelled words will be reported. You can add words to be
 ignored to ``ignore_words.txt``
 
 
 Documentation
 -------------
-Documentation for DPF-Core is generated from three sources:
+Documentation for PyDPF-Core is generated from three sources:
 
 - Docstrings from the classes, functions, and modules of ``ansys.dpf.core`` using `sphinx.ext.autodoc <https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html>`_.
```

Comment:
Wasn't sure if sphinx.ext.autodoc was still  correct here.

Reply:
This is correct. We use this extension in all the major pyansys repositories.